### PR TITLE
Add decoupled Python3 OMERO.web

### PR DESCRIPTION
**Pull request by manics**
Renamed `tasks/web-install.yml → tasks/web-install-py2.yml` and made a few minor cleanups, but the existing behaviour should be unchanged.
New variables (Python 3 only, otherwise ignored):
- `omero_web_python3`
- `omero_web_python_addons`
- `ome.omero_web_apps` is merged into this role: https://github.com/ome/ansible-role-omero-web-apps/pull/7 so all parameters from that role are now handled here

To avoid chaining python3 changes through the whole stack of roles this is self-contained.

Breaking changes when switching to Python 3 are listed in `CHANGES.md`

Closes https://github.com/ome/ansible-role-omero-web/issues/17
Closes https://github.com/ome/ansible-role-omero-web/issues/24
Closes https://github.com/ome/ansible-role-omero-web/issues/20

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [ANSIBLE-ROLES-merge#875](https://ci.openmicroscopy.org/job/ANSIBLE-ROLES-merge/875/). See the [console output](https://ci.openmicroscopy.org/job/ANSIBLE-ROLES-merge/875/consoleText) for more details.
Possible conflicts:
  - PR #18 manics 'Ensure config is flushed and systemd is setup before nginx'
    - tasks/web-nginx.yml

--conflicts

File: defaults/main.yml [hunk 1]
```diff
@@ -31,11 +31,40 @@ omero_web_config_set: {}
...
+  - https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.2.1/zeroc_ice-3.6.5-cp36-cp36m-linux_x86_64.whl
+  - omero-py==5.6.dev2
+  - omero-web==5.6.dev3
+  - redis
+
+# Path to virtualenv
+omero_web_virtualenv_basedir: >-
+  {{
+    omero_web_python3 | ternary(
+      omero_web_basedir + '/venv3',
```
Comment (joshmoore) [inline defaults/main.yml]: I understand a benefit of using a new directory is to have a cleaner migration, but there's a good possibility that end-user scripts will need updating.

File: README.md [hunk 2]
```diff
@@ -35,6 +36,24 @@ OMERO.web systemd configuration
 - `omero_web_systemd_after`: A list of strings with additional service names to appear in systemd unit file "After" statements. Default empty/none.
 - `omero_web_systemd_requires`: A list of strings with additional service names to appear in systemd unit file "Requires" statements. Default empty/none.
 
+Python 3 only parameters
+- `omero_web_python_addons`: List of additional Python packages to be installed into virtualenv
```
Comment (mtbc) [inline README.md]: :+1:

**Comment by joshmoore** [comment]
Started worked in https://github.com/manics/ansible-role-omero-web/pull/1 -- a bit less clear as a variable since the file didn't exist previously.

**Comment by joshmoore** [comment]
This has been tested with https://github.com/ome/omero-web-docker/pull/42 and released as docker image `openmicroscopy/omero-web:5.6.0-m3-1`. Merging.

A couple of issues have been opened for follow-ups. If we had dev releases on galaxy, I'd propose an immediate 3.0.0.dev1 / 3.0.0-m1 release. Without, I'm not sure if we want the issues handled first.

cc: @manics @sbesson

**Pull request merged**